pkg/panels/kubernetes/apiserver.go

```diff
@@ -0,0 +1,428 @@
+package kubernetes
+
+import (
+	"github.com/perses/community-dashboards/pkg/dashboards"
+	"github.com/perses/community-dashboards/pkg/promql"
+	"github.com/perses/perses/go-sdk/panel"
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	markdown "github.com/perses/plugins/markdown/sdk/go"
+	"github.com/perses/plugins/prometheus/sdk/go/query"
+	statPanel "github.com/perses/plugins/statchart/sdk/go"
+	timeSeriesPanel "github.com/perses/plugins/timeserieschart/sdk/go"
+
+	commonSdk "github.com/perses/perses/go-sdk/common"
+)
+
+func APIServerSLONotice(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
+	return panelgroup.AddPanel("Notice",
+		markdown.Markdown("Notice",
+			markdown.Text("The SLO (service level objective) and other metrics displayed on this dashboard are for informational purposes only.")),
+	)
+}
+
+func APIServerAvailability(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
+	return panelgroup.AddPanel("Availability (30d) > 99.000%",
+		panel.Description("How many percent of requests (both read and write) in 30 days have been answered successfully and fast enough?"),
+		statPanel.Chart(
+			statPanel.Calculation(commonSdk.LastCalculation),
+			statPanel.Format(commonSdk.Format{
+				Unit:          string(commonSdk.PercentUnit),
+				DecimalPlaces: 3,
+			}),
+			statPanel.ValueFontSize(50),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"apiserver_request:availability30d{verb='all', cluster=~'$cluster'}",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+	)
+}
+
+func APIServerErrorBudget(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
```

Comment:
Would this be a timeseriesPanel? The upstream json seems to be,
```
  "targets": [
            {
               "datasource": {
                  "type": "prometheus",
                  "uid": "${datasource}"
               },
               "expr": "100 * (apiserver_request:availability30d{verb=\"all\", cluster=\"$cluster\"} - 0.990000)",
               "legendFormat": "errorbudget"
            }
         ],
         "title": "ErrorBudget (30d) > 99.000%",
         "type": "timeseries"
      },
```

Reply:
Yes it was, it kind of made more sense in my view to have all the slo data in stat format, but I adjusted to the original format. Thanks